### DISCUSSION TITLE
Cant connect to docker ip after installing tiny windows 11 and .config file already exists

### DISCUSSION THREAD

**Discussion by BlueFox1616** [winapps-org/winapps#534] [Support]
I left my tiny windows eleven installation to run over night and in the morning it had lost connection i tried refreshing the site stoping and starting docker but i am unable to access the site also in my system a .config file already exists and when i put winapps there both the docker throws an error and the bin/winapps check command

**Comment by LDprg** [comment]
you need to set the ip in the config to local host

**Comment by BlueFox1616** [comment]
I have set it to RDP_IP="127.0.0.1" in the winapps.conf file im i missing an option or do you mean something else
